src/agent_dashboard_helper.js

```diff
@@ -20,7 +24,16 @@ function renderPendingAndTodaysTrips(trips, destinations) {
   let pendingTrips = dashboardHelper.getPendingTrips(trips);
   renderAgentTrips(pendingTrips, destinations);
   dashboardHelper.renderTripsHeader("Today's");
+  document.getElementsByClassName("container")[1].id = "todays-trips";
   let todaysTrips = dashboardHelper.getTodaysTrips(trips);
+  if (todaysTrips.todaysTrips.length === 0) {
+    let noTravelersTextNode = document.createTextNode(
+      "There are no travelers on Trips Today"
+    );
+    let noTravelersH1 = document.createElement("h1");
+    noTravelersH1.appendChild(noTravelersTextNode);
+    document.querySelectorAll("section")[1].appendChild(noTravelersH1);
+  }
```

Comment:
This is a pretty long addition to this function.

Maybe you could d have this whole thing be a helper?

call something like `checkForNoTravelers(todaysTrips);` or whatever